sleap/gui/commands.py

```diff
@@ -2120,6 +2120,12 @@ def do_action(cls, context: CommandContext, params: dict):
             context.app,
         )
 
+        if (
+            params["target"] == "current video"
+        ):  # checks if current video is selected in gui
+            params["videos"] = [context.state["video"]]
```

Comment:
Edge cases: `context.state["video"]` is `None` and `len(context.labels.videos) == 0`. Can we include a check for that here, i.e., use `context.state["video"]` if not `None` else use `context.labels.videos[0]` if `len(context.labels.videos) > 0`. Otherwise, print an error message in the terminal saying no videos were found to generate suggestions on.

Additionally, we can [disable the GUI button](https://github.com/talmolab/sleap/blob/adcc56ed6bdbf1bea10e7e843934a73453322d81/sleap/gui/app.py#L1172) "Generate Suggestions" if `len(context.labels.videos) == 0`.